radixdlt-java/src/test/java/com/radixdlt/client/core/fungible/FungibleParticleTransitionerTest.java

```diff
@@ -50,11 +52,23 @@ public void when_creating_a_transition_from_one_to_one__then_transitioned_should
 	}
 
 	@Test
-	public void when_creating_a_transition_from_two_to_one__then_transitioned_should_be_one_migrated_should_be_one_removed_should_be_a_two() {
+	public void when_creating_a_transition_from_two_to_one__then_transitioned_should_be_one_migrated_should_be_one_removed_should_be_a_two()
```

Comment:
On a completely unrelated note, I'm wondering if this way of naming tests is actually a net benefit. Ironically makes it quite hard to read what the test does at a quick glance.

Reply:
It actually works really well for me

Reply:
Maybe the benefit lies "in the eye of the beholder". I personally find it so much easier to understand the expected outcome of someone else's test case (or my own after a couple of weeks) when written like this or in a similar fashion. It also helps me focus on testing what I claim I'm testing.

When testing a bigger feature I usually also group the test cases within the test class and wrap the groups with an easily consumable short comment on what's being grouped.